main.py

```diff
@@ -27,16 +27,28 @@ def multi_invite(org: str, a_users: list):
         uid = gh.get_user_id(user)
 
         if uid is None:
-            cprint(f"[red]: user {user} does not exists")
+            cprint(f"[red]: user {user} does not exist or is already a member of the organization")
+            continue
+
+        # Check if user is already a member of the organization
+        if gh.is_user_member_of_org(org, uid):
+            cprint(f"[yellow]: user {user} is already a member of the organization")
             continue
 
         cprint(f"[green]: sending invite to {user}.")
         inv = gh.invite_user_org(org, uid)
 
         if inv.status_code == 201:
             cprint(f"[green]: invite sent to {user}.")
+        elif inv.status_code == 422:  # 422 Unprocessable Entity
+            error_response = inv.json()
+            if any(error['code'] == 'unprocessable' for error in error_response.get('errors', [])):
+                cprint(f"[yellow]: user {user} is already invited or a member of the organization")
```

Comment:
![image](https://github.com/GDSCParulUniversity/GitHub-multi-user-invite/assets/68287637/7ae1ce30-75f1-4007-bb21-8710be3376aa)

GitHub won't say anything if the user got invited already and if the response is pending.

```suggestion
                cprint(f"[yellow]: user {user} is already invited.")
```